libs/blocks/hero-marquee/hero-marquee.css

```
@@ -0,0 +1,581 @@
+:root {
+  --s-min-height: 248px; /* 360px */
+  --m-min-height: 448px; /* 560px */
+  --l-min-height: 588px; /* 700px */
+  
+  /* 112 = 56px default padding */
+}
+
+.hero-marquee {
+  padding: var(--spacing-xl) 0;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  gap: var(--spacing-m);
+  width: 100%;
+  overflow: hidden;
+  min-height: var(--m-min-height);
+  justify-content: center;
+  align-items: center;
+}
+
+.hero-marquee.s-min-height { min-height: var(--s-min-height);}
+.hero-marquee.l-min-height { min-height: var(--l-min-height);}
```

Linter might complain
```suggestion
.hero-marquee.s-min-height { min-height: var(--s-min-height); }
.hero-marquee.l-min-height { min-height: var(--l-min-height); }
```